### PR TITLE
🆕🌍Adding Yandex Maps geocoder

### PR DESCRIPTION
**Pull request by mekegi**
https://yandex.com/maps/
tech documentation
https://tech.yandex.com/maps/doc/geocoder/desc/concepts/About-docpage/

### CODE REVIEW

File: yandex/geocoder.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
+// Package yandex is a geo-golang based Yandex Maps Location API
+// https://tech.yandex.com/maps/doc/geocoder/desc/concepts/About-docpage/
```
Comment (codingsince1985) [inline yandex/geocoder.go]: Can you please remove this line, and add below in README after line 27?
+ [Yandex](https://tech.yandex.com/maps/doc/geocoder/desc/concepts/About-docpage/)

Comment (mekegi) [reply yandex/geocoder.go]: @codingsince1985 done

File: yandex/geocoder.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+	}
+
+	return parseYandexResult(r.Response.GeoObjectCollection.FeatureMember[0]), nil
+}
+
+func parseYandexResult(r *yandexFeatureMember) *geo.Address {
+	addr := &geo.Address{}
+	res := r.GeoObject.MetaDataProperty.GeocoderMetaData
+
+OuterLoop:
```
Comment (kruftmeister) [inline yandex/geocoder.go]: You don't the `continue OuterLoop`. Just `continue` will do

Comment (mekegi) [reply yandex/geocoder.go]: done

**Comment by codingsince1985** [comment]
Great contribution, @mekegi!

**Pull request merged**